__tests__/Itinerary.test.js

```diff
@@ -0,0 +1,18 @@
+const Itinerary = require("../src/Itinerary.js");
```

Comment:
Ive commented about formatting on other files so wont repeat the same info here, but this should be reformatted so it has 2 spaces of indentation and the extra consecutive blank lines removed